Makefile

```diff
@@ -98,10 +98,10 @@ check-env-build:
 		exit 1; \
 	fi
 
-requirements.txt:
```

Comment:
it should instead be `requirements.txt: uv.lock`

`uv.lock` being the input used to generate `requirements.txt`

Reply:
done